test/should_fail/record_field.erl

```diff
@@ -1,9 +1,13 @@
 -module(record_field).
 
--export([f/1]).
+-export([f/1, g/1]).
 
 -record(rec, { apa :: integer()}).
 
 -spec f(#rec{}) -> boolean().
 f(A) ->
     A#rec.apa.
+
+-spec g(#rec{}) -> pos_integer().
+g(A) ->
+    A#rec.apa.
```

Comment:
Why should this fail? `apa` is the first record element, i.e. the 2nd tuple element, isn't it?